examples/tests/stack_size_test02/Makefile

```diff
@@ -7,6 +7,9 @@ TOCK_USERLAND_BASE_DIR = ../../..
 C_SRCS := $(wildcard *.c)
 
 STACK_SIZE := 9001
+# Required to work on RISC-V boards
+APP_HEAP_SIZE    = 4096
+KERNEL_HEAP_SIZE = 2048
```

Comment:
So without these changes you see a different failure instead of a stack error? I am confused because this app does almost nothing at all aside from attempting to set a too-large stack size -- why would the kernel need a 2kB heap for this? Is that just for the console driver?